plugins/destination/sqlite/client/migrate.go

```diff
@@ -148,7 +148,7 @@ func (c *Client) createTableIfNotExist(_ context.Context, table *schema.Table) e
 	sb.WriteString(")")
 	_, err := c.db.Exec(sb.String())
 	if err != nil {
-		return fmt.Errorf("failed to create table %s: %w", table.Name, err)
+		return fmt.Errorf("failed to create table with '%s': %w", sb.String(), err)
```

Comment:
Would use `%q` instead of `'%s'` so that it's properly quoted
```suggestion
		return fmt.Errorf("failed to create table with %q: %w", sb.String(), err)
```